.github/CODEOWNERS

```diff
@@ -2,3 +2,2118 @@
+modules/nf-core/methyldackel/mbias/** @phue
+modules/nf-core/mykrobe/predict/** @rpetit3
+modules/nf-core/methyldackel/extract/** @phue
+modules/nf-core/sratools/fasterqdump/** @Midnighter
+modules/nf-core/abacas/** @joseespinosa @drpatelh
+modules/nf-core/antismash/antismashlite/** @jasmezz
+modules/nf-core/mitohifi/findmitoreference/** @verku
+modules/nf-core/hmmer/hmmsearch/** @Midnighter
+modules/nf-core/hmmer/eslreformat/** @erikrikarddaniel
+modules/nf-core/hmmer/hmmfetch/** @erikrikarddaniel
+modules/nf-core/hmmer/hmmbuild/** @erikrikarddaniel
+modules/nf-core/hmmer/hmmalign/** @erikrikarddaniel @jfy133
+modules/nf-core/hmmer/eslalimask/** @erikrikarddaniel
+modules/nf-core/mitohifi/mitohifi/** @ksenia-krasheninnikova
+modules/nf-core/antismash/antismashlitedownloaddatabases/** @jasmezz
+modules/nf-core/minimap2/align/** @heuermh @sofstam @sateeshperi @jfy133
+modules/nf-core/sentieon/varcal/** @asp8200
+modules/nf-core/minimap2/index/** @yuukiiwa @drpatelh
+modules/nf-core/gvcftools/extractvariants/** @nvnieuwk
+modules/nf-core/sentieon/dnascope/** @ramprasadn
+modules/nf-core/sentieon/tnhaplotyper2/** @asp8200
+modules/nf-core/sentieon/applyvarcal/** @assp8200
+modules/nf-core/sentieon/dedup/** @asp8200
+modules/nf-core/sentieon/haplotyper/** @asp8200
+modules/nf-core/sentieon/gvcftyper/** @asp8200
+modules/nf-core/sentieon/readwriter/** @ramprasadn
+modules/nf-core/sentieon/dnamodelapply/** @ramprasadn
+modules/nf-core/sentieon/wgsmetrics/** @ramprasadn
+modules/nf-core/graphtyper/vcfconcatenate/** @zachary-foster
+modules/nf-core/sentieon/bwamem/** @asp8200
+modules/nf-core/sentieon/bwaindex/** @drpatelh @maxulysse
+modules/nf-core/graphtyper/genotype/** @zachary-foster
+modules/nf-core/sentieon/tnscope/** @ramprasadn
+modules/nf-core/ariba/run/** @rpetit3
+modules/nf-core/ariba/getref/** @rpetit3
+modules/nf-core/sentieon/datametrics/** @ramprasadn
+modules/nf-core/meningotype/** @rpetit3
+modules/nf-core/maxbin2/** @jfy133
+modules/nf-core/freebayes/** @maxibor @FriederikeHanssen @maxulysse
+modules/nf-core/sickle/** @BioInf2305
+modules/nf-core/segemehl/index/** @BarryDigby
+modules/nf-core/amplify/predict/** @louperelo
+modules/nf-core/segemehl/align/** @BarryDigby
+modules/nf-core/entrezdirect/xtract/** @alxndrdiaz
+modules/nf-core/entrezdirect/esearch/** @alxndrdiaz
+modules/nf-core/entrezdirect/esummary/** @alxndrdiaz
+modules/nf-core/smoove/call/** @scorreard @nvnieuwk
+modules/nf-core/medaka/** @avantonder
+modules/nf-core/mash/sketch/** @thanhleviet
+modules/nf-core/iphop/predict/** @CarsonJM
+modules/nf-core/mash/dist/** @rpetit3
+modules/nf-core/abricate/summary/** @rpetit3
+modules/nf-core/seroba/run/** @rpetit3
+modules/nf-core/mash/screen/** @mahesh-panchal
+modules/nf-core/sequencetools/pileupcaller/** @TCLamnidis
+modules/nf-core/abricate/run/** @rpetit3
+modules/nf-core/fastqc/** @drpatelh @grst @ewels @FelixKrueger
+modules/nf-core/sgdemux/** Samantha White: @samfulcrum
```

Comment:
@Emiller88 , what's going on in this line?

Reply:
Good catch, I think just a typo with the format from @samfulcrum